.changeset/short-aliens-invite.md

```diff
@@ -0,0 +1,18 @@
+---
+'@backstage/plugin-api-docs': minor
+'@backstage/plugin-techdocs': minor
+---
+
+Remove explicit dependency on `isomorphic-form-data`.
+
+This explicit dependency was added to address [an issue](https://github.com/swagger-api/swagger-ui/issues/7436) in the
+dependency `swagger-ui-react`. That [issue has since been resolved](https://github.com/swagger-api/swagger-ui/issues/7436#issuecomment-889792304),
+and `isomorphic-form-data` no longer needs to be declared.
+
+Additionally, this changeset updates the `swagger-ui-react` dependency to version `5.19.0` or higher, which includes
+[compatibility](https://github.com/swagger-api/swagger-ui?tab=readme-ov-file#compatibility) with the latest versions of
+the OpenAPI specification.
+
+In the course of this upgrade, there were a handful of type errors that cropped up in `@backstage/plugin-techdocs`,
+those have been remediated through the addition of some type guards and loosening of some type signatures from `Element`
+to `Node`
```

Comment:
Overall these TechDocs changes seem odd to be here. Would be good to have better clarity on exactly why they are needed. Not sure why changing `api-docs` would cause this. Maybe rebase, you a bit behind but not that much 🤔

Reply:
They seemed odd to me as well, but my best guess is that in the course of updating these deps, some transitives moved and had some changes. Specifically, these techdocs updates are around `dompurify` which moved up to version: `3.2.4`. 

I didn't investigate what might be different between dompurify 3.1.6 and 3.2.4, but that was the source of the angry types (if I recall correctly, it was ONLY types, the tests ran just fine)